capitals.rb

```diff
@@ -51,102 +51,142 @@
-    capital: "Richmond"
-}, {
-    name: "Washington",
-    capital: "Olympia"
-}, {
-    name: "West Virginia",
-    capital: "Charleston"
-}, {
-    name: "Wisconsin",
-    capital: "Madison"
-}, {
-    name: "Wyoming",
-    capital: "Cheyenne"
+     name: "Louisiana",
+     capital: "Baton Rouge"
+}, {
+     name: "Maine",
+     capital: "Augusta"
+ }, {
+     name: "Maryland",
+     capital: "Annapolis"
+ }, {
+     name: "Massachusetts",
+     capital: "Boston"
+ }, {
+     name: "Michigan",
+     capital: "Lansing"
+ }, {
+     name: "Minnesota",
+     capital: "St. Paul"
+ }, {
+     name: "Mississippi",
+     capital: "Jackson"
+ }, {
+     name: "Missouri",
+     capital: "Jefferson City"
+ }, {
+     name: "Montana",
+     capital: "Helena"
+ }, {
+     name: "Nebraska",
+     capital: "Lincoln"
+ }, {
+     name: "Nevada",
+     capital: "Carson City"
+ }, {
+     name: "New Hampshire",
+     capital: "Concord"
+ }, {
+     name: "New Jersey",
+     capital: "Trenton"
+ }, {
+     name: "New Mexico",
+     capital: "Santa Fe"
+ }, {
+     name: "New York",
+     capital: "Albany"
+ }, {
+     name: "North Carolina",
+     capital: "Raleigh"
+ }, {
+     name: "North Dakota",
+     capital: "Bismarck"
+ }, {
+     name: "Ohio",
+     capital: "Columbus"
+ }, {
+     name: "Oklahoma",
+     capital: "Oklahoma City"
+ }, {
+     name: "Oregon",
+     capital: "Salem"
+ }, {
+     name: "Pennsylvania",
+     capital: "Harrisburg"
+ }, {
+     name: "Rhode Island",
+     capital: "Providence"
+ }, {
+     name: "South Carolina",
+     capital: "Columbia"
+ }, {
+     name: "South Dakota",
+     capital: "Pierre"
+ }, {
+     name: "Tennessee",
+     capital: "Nashville"
+ }, {
+     name: "Texas",
+     capital: "Austin"
+}, {
+     name: "Utah",
+     capital: "Salt Lake City"
+}, {
+     name: "Vermont",
+     capital: "Montpelier"
+}, {
+     name: "Virginia",
+     capital: "Richmond"
+}, {
+     name: "Washington",
+     capital: "Olympia"
+}, {
+     name: "West Virginia",
+     capital: "Charleston"
+}, {
+     name: "Wisconsin",
+     capital: "Madison"
+}, {
+     name: "Wyoming",
+     capital: "Cheyenne"
 }]
+
+print "It's time to learn state capitals!\nWrite the capital of the states as they appear.\n"
+
+states.each do |state|
+state[:numR] = 0
+state[:numW] = 0
+
+end
+
+  loop do
+    states.shuffle { |state, capital| puts state }.each do |state|
+      puts "#{state[:name]}"
+
+    userAns = gets.chomp
+
+    if userAns.upcase == "#{state[:capital]}".upcase
+      puts "Nice work! You know this one."
+      state[:numR] +=1
+
+      totalC = state[:numR] + state[:numW]
+      puts "You've answered that correctly #{state[:numR]} out of #{totalC} times."
+
+    else
+      puts "The correct answer is #{state[:capital]}. We'll come back to this one."
+      state[:numW] +=1
+
+      totalC = state[:numR] + state[:numW]
+      puts "You've answered that correctly #{state[:numR]} out of #{totalC} times."
+
+    end
+  end
+
```

Comment:
I think you're not getting the ordering and cycling through again because your break statement is outside the loop, causing an infinite loop.